### ISSUE TITLE
Would you like to update  this to MAUI?

### ISSUE THREAD

**Issue by ymqc** [johnthiriet/Xamarin.Android.Huawei.Hms#6]
Good job, Thanks for your work,  will this support MAUI in some day?

**Comment by johnthiriet** [comment]
Hello,

These are just bindings over the native HMS plugins.

MAUI is the Xamarin.Forms equivalent of the .NET 6 world let's say, therefore, it is done to be independent from the underlying platform.

Bindings are not independent for the platform and are not "Xamarin.Forms" plugins. So no they will not come to MAUI.

Those bindings should work fine with the latest version of Xamarin.Native. I did not test them in a fully .NET 6 mobile solution and I do not plan to do it for now but I see no reason why the nugets I created would not work in a fully .NET 6 mobile.

**Comment by johnthiriet** [comment]
Hello,

These are just bindings over the native HMS plugins.

MAUI is the Xamarin.Forms equivalent of the .NET 6 world let's say, therefore, it is done to be independent from the underlying platform.

Bindings are not independent for the platform and are not "Xamarin.Forms" plugins. So no they will not come to MAUI.

Those bindings should work fine with the latest version of Xamarin.Native. I did not test them in a fully .NET 6 mobile solution and I do not plan to do it for now but I see no reason why the nugets I created would not work in a fully .NET 6 mobile.
